opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java

```diff
@@ -13,11 +13,13 @@
  */
 package io.opentracing.mock;
 
+import io.opentracing.References;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
isn't this only available from 1.7+?

Reply:
Yes. I thought the compilation source and target were 1.7 since it is set in the pom file.

Reply:
ah, ok - the api module is 1.6, the rest are 1.7, so no new change is introduced here